src/app/components/app-build-details/app-build-details.component.html

```diff
@@ -24,7 +24,7 @@ <h1 class="bold">
                       </span>
                     </h1>
                   </div>
-                  <div class="column is-2" *ngIf="build?.hasPermission">
+                  <div class="column is-2" *ngIf="build?.hasPermission" [class.is-offset-2]="status === 'success' || status === 'failed'">
```

Comment:
this won't work as expected unfortunatelly, `.is-offset-2` gives column `.is-2` space on each side, so that means `.is-4` if we say so.